pkg/sql/logictest/tests/local/local_test: TestLogic_lookup_join_local failed

pkg/sql/logictest/tests/local/local_test.TestLogic_lookup_join_local [failed](https://tanzanite.cluster.engflow.com/invocations/default/66eef19a-69b0-406c-aa14-39903901cf98?testReportRun=2&testReportShard=27&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [067e48d29b9093038f6fcf2074cd761ffdcd4fe2](https://github.com/cockroachdb/cockroach/commits/067e48d29b9093038f6fcf2074cd761ffdcd4fe2):

Fatal error:

```
panic: test timed out after 1h16m37s
running tests:
	TestLogic_lookup_join_local (1h15m10s)

```
Stack: 

```
goroutine 886102 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestLogic_lookup_join_local
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_lookup_join_local204621731
    test_log_scope.go:81: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=27</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_lookup_join_local.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-36772

--- a/pkg/sql/logictest/testdata/logic_test/lookup_join_local
+++ b/pkg/sql/logictest/testdata/logic_test/lookup_join_local
@@ -1,5 +1,7 @@
 # LogicTest: local !metamorphic-batch-sizes
 
+skip under race
+
 # This test verifies that the row container used by the join reader spills to
 # disk in order to make room for the non-spillable internal state of the
 # processor.

